cloud/google/update_mirror.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+if [ "$1" == "" ]; then
+  echo "Usage $0: <config-file>"
+  exit 1
+fi
+source ${DIR}/util.sh
+source ${DIR}/config.sh $1
+
+set -e
+GCLOUD="gcloud"
+
+Header "Updating mirror instances..."
+# TODO(alcutter): Wait for each task (or zone?) to be healthy before
+# proceeding.
+for i in `seq 0 $((${MIRROR_NUM_REPLICAS} - 1))`; do
+  echo "Updating ${MIRROR_MACHINES[${i}]}"
+  gcloud compute ssh ${MIRROR_MACHINES[${i}]} \
+      --zone ${MIRROR_ZONES[${i}]} \
+      --command \
+          'sudo docker pull gcr.io/'${PROJECT}'/super_mirror:test &&
```

Comment:
`/MIRROR_/LOG_/` too.
But the reasons I didn't factor it out are:
 - passing arrays is painful in bash, though I could just copy the contents of the `LOG_*` and `MIRROR_*` to e.g. `NODE_*` before calling, but
 -  when the `TODO`s are `TODONE`, there could be different checks necessary depending on you're updating (especially if that could also be an `etcd` instance.), again I guess you could define a function and then pass in the command to call it which gets `eval`'d  or something, but you have to question whether it's worth it to dedupe this pretty small loop...